src/modules/taskStatus.js

```diff
@@ -0,0 +1,31 @@
+// let todos = JSON.parse(localStorage.getItem("todos")) || [];
+// save todo
+
+import { todoInput, todos } from '../app.js';
+
+export default function saveTodo() {
+  const todoValue = todoInput.value;
+
+  // check if todo id empty
+  const isEmpty = todoValue === '';
+  // check if duplicat
+  const isDuplicate = todos.some(
+    (todo) => todo.value.toUpperCase() === todoValue.toUpperCase(),
+  );
+
+  if (isEmpty) {
+    alert('input a todo');
+  } else if (isDuplicate) {
+    alert('Todo Already Exist');
+  } else {
+    const todo = {
+      value: todoValue,
+      checked: false,
+      color: `#${Math.floor(Math.random() * 16777215).toString(16)}`,
+    };
+    todos.push(todo);
+    todoInput.value = '';
+    console.log(todos);
+  }
+}
+console.log('hi');
```

Comment:
kindly it is not good to push a console.log to your GitHub repository since this can result in errors while running the project.

Reply:
Good point.

Reply:
Do you have any suggestion for me how to fix this linter error

Reply:
Please run linters locally on your computer and try auto-fix if they seem to be challenging.